tests/sentry/api/serializers/test_incident.py

```diff
@@ -35,8 +35,8 @@ def test_simple(self):
         assert result["dateDetected"] == incident.date_detected
         assert result["dateCreated"] == incident.date_added
         assert result["dateClosed"] == incident.date_closed
-        assert len(result["eventStats"]["data"]) == 62
-        assert [data[1] for data in result["eventStats"]["data"]] == [[]] * 62
+        assert len(result["eventStats"]["data"]) == 61
```

Comment:
@iProgramStuff @wedamija @billyvg  Can you folks verify if this is right? I'm not familiar with the internals of incidents.

Reply:
Looks like this is right.

Reply:
Yeah seems consistent with the other changes.